app/models/event_attendance.rb

```diff
@@ -35,7 +35,7 @@ def update_filter_cache
 
   # Class method to use along with other named scopes to limit results to a specific audience group.
   def self.audience(audience_name = Person)
-    where(["(audience = :audience) OR (people.type = :audience AND audience = NULL)", {:audience => audience_name.to_s.classify}])
+    joins(:person).where(["(audience = :audience) OR (people.type = :audience AND audience IS NULL)", {:audience => audience_name.to_s.classify}])
```

Comment:
Line is too long. [146/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.